listenbrainz/listenstore/timescale_listenstore.py

```diff
@@ -183,8 +183,6 @@ def insert(self, listens):
 
         query = """INSERT INTO listen (listened_at, track_name, user_name, user_id, data)
                         VALUES %s
-                   ON CONFLICT (listened_at, track_name, user_id)
```

Comment:
urgh, I thought I undid this from the commit, this is from some other stuff I did, should definitely not be here